src/Google.Api.Gax.Grpc/ApiCall.cs

```diff
@@ -94,5 +94,17 @@ public sealed class ApiCall<TRequest, TResponse>
                 _asyncCall.WithRetry(clock, scheduler, null),
                 _syncCall.WithRetry(clock, scheduler, null),
                 BaseCallSettings);
+
+        /// <summary>
+        /// Construct a new <see cref="ApiCall{TRequest, TResponse}"/> that applies an overlay to
+        /// the underlying <see cref="CallSettings"/>.
```

Comment:
Possibly document the priority, i.e. if a value exists in both the original and the overlay, the overlay takes priority?

Reply:
Done